src/NuGet.Services.AzureSearch/DocumentUtilities.cs

```diff
@@ -29,6 +29,11 @@ public static string GetHijackDocumentKey(string packageId, string normalizedVer
             return EncodeKey($"{lowerId}/{lowerVersion}");
         }
 
+        public static double GetDownloadScore(double totalDownloadCount)
+        {
+            return Math.Sqrt(totalDownloadCount) / 220;
```

Comment:
The `/ 220` bit doesn't affect the final score, however, it makes download scores easier to reason about as they range from 0 to <100.

Reply:
sqrt(500_000_000)/220 == ~101

Reply:
This needs a code comment, including where the 500 million comes from.

Reply:
so download count of 48400 will give a score of 1. Any ideas on how many packages currently exceed the total download count of 48400? Given that aggregation function is a sum(?) other weights might boost results for packages with download count of less than 48K

Reply:
@shishirx34 Yup, this current algorithm isn't great at low download counts. Right now, the tokenized package id field's score has the strongest affect on the overall score at low downloads. This will need to be addressed in the future.